src/Configurator/DockerComposeConfigurator.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Flex\Configurator;
+
+use Symfony\Flex\Lock;
+use Symfony\Flex\Recipe;
+
+/**
+ * Adds services and volumes to docker-compose.yaml file.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class DockerComposeConfigurator extends AbstractConfigurator
+{
+    public function configure(Recipe $recipe, $config, Lock $lock, array $options = [])
+    {
+        $installDocker = $this->composer->getPackage()->getExtra()['symfony']['docker'] ?? false;
+        if (!$installDocker) {
+            return;
+        }
+
+        $dockerCompose = $this->options->get('root-dir').'/docker-compose.yaml';
```

Comment:
We need to support `docker-compose.yml` as well, right?

Reply:
We could also support the `COMPOSE_PROJECT_NAME` env var: https://docs.docker.com/compose/reference/envvars/#compose_project_name

Reply:
How could we use `COMPOSE_PROJECT_NAME`?

Reply:
To distinguish containers of multiple projects running at the same time? You would set this env var to the name defined in `composer.json`? Actually it must be set before running `docker-compose`, so it should be in Symfony CLI I guess.

Reply:
`docker-compose.yml` support added

Reply:
Docker Compose always reads `.env`. Special env vars like `COMPOSE_PROJECT_NAME` can be set in there. But also, any env var defined in `.env` can be used for interpolation in `docker-compose.yml`